Documentation/contributing/development/reviewers_committers/duties.rst

```diff
@@ -0,0 +1,49 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _weekly_duties:
+
+Weekly Duties
+=============
+
+Some members of the Cilium organization will have rotational duties that change
+every week.
+
+Release Managers
+----------------
+
+Release managers take care of the patch releases for each supported stable
+branch of Cilium. They typically coordinate in #launchpad on Slack.
```

Comment:
A link would be good here, or more specificity
```suggestion
branch of Cilium. They typically coordinate in #launchpad on Cilium Slack.
```

Reply:
I realise links to Cilium Slack throughout the docs are a mess (I count 13 occurrences of the URL). I'll harmonise them in a follow-up PR.

Reply:
https://github.com/cilium/cilium/pull/27346